test/base_pg_regress.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Is there really no way to unify this script with the existing `pg_regress.sh` that we already have? I mean they are both calling the same program in the end so any options that need to be different should be possible to set via, e.g., ENV variables. Now we need to maintain two scripts that do similar things.

Reply:
Of course they can be merged. I opted for different scripts as I though that this would be a special case called from a different test harness and I wanted all this extra code to be isolated. I can change it if it is important.

Reply:
It just seems there is a lot of overlap and I'd rather maintain one script than two. But if you think that this is such a special case that it would bloat the single script or make it difficult to maintain, then fine.